languages/python/sqlalchemy-oso/sqlalchemy_oso/oso.py

```diff
@@ -0,0 +1,55 @@
+from oso import Oso, OsoError
+from .auth import register_models
+from .roles2 import OsoRoles
+
+
+class SQLAlchemyOso(Oso):
+    """The central object to manage application policy state, e.g.
+    the policy data, and verify requests when using Oso with SQLAlchemy.
+
+    Supports SQLAlchemy-specific functionality, including data filtering and role management.
+
+    Accepts a SQLAlchemy declarative_base on initialization, which is used to register
+    all relevant SQLAlchemy models with Oso.
+
+    >>> from sqlalchemy-oso import SQLAlchemyOso
```

Comment:
[Enable doctests in PyTest](https://docs.pytest.org/en/stable/doctest.html) so that this example is tested. Right now it's incorrect (`from sqlalchemy-oso` should be `from sqlalchemy_oso`)

Reply:
Got it. I can update docs--I'll coordinate with Gabe about GitClub.